src/EditorFeatures/VisualBasicTest/ImplementInterface/ImplementInterfaceTests.vb

```diff
@@ -4372,39 +4394,36 @@ End Interface",
  index:=1)
         End Function
 
-        Private Shared Function DisposePattern(disposeMethodModifiers As String, Optional simplifySystem As Boolean = True) As String
+        Private Shared Function DisposePattern(
+                disposeMethodModifiers As String,
+                Optional simplifySystem As Boolean = True,
+                Optional disposeField As String = "disposedValue",
+                Optional gcPrefix As String = "",
+                Optional disposeMethodAccessibility As String = "Protected") As String
             Dim code = $"
-#Region ""IDisposable Support""
-    Private disposedValue As Boolean ' {FeaturesResources.To_detect_redundant_calls}
-
-    ' IDisposable
-    Protected {disposeMethodModifiers}Sub Dispose(disposing As Boolean)
-        If Not disposedValue Then
+    {disposeMethodAccessibility} {disposeMethodModifiers}Sub Dispose(disposing As Boolean)
+        If Not {disposeField} Then
+            ' {FeaturesResources.TODO_colon_dispose_managed_state_managed_objects}
```

Comment:
❗️  I believe the relocation of this comment will negatively impact how people implement the method. It should remain inside the `If` statement for both C# and Visual Basic.

Reply:
Ok.  Can def makethat happen.

Reply:
Done.